### PR TITLE
Release notes for 3/5/2020

### PR DESCRIPTION
**Pull request by agau4779**

### CODE REVIEW

File: firestore-bigquery-export/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## Version 0.1.2
+
+fixed - Added "IF NOT EXISTS" to safely run script multiple times
+fixed - Updated BigQuery library
```
Comment (jhuleatt) [inline firestore-bigquery-export/CHANGELOG.md]: Can you please add a reference to #197 ?

Suggested change to the message:

```
Updated BigQuery dependency in fs-bq-import-collection package.json (#197)
```

cc @IanWyszynski

Comment (rachelsaunders) [reply firestore-bigquery-export/CHANGELOG.md]: Is this suggestion for the first or the second bullet?

Comment (jhuleatt) [reply firestore-bigquery-export/CHANGELOG.md]: the second bullet. first bullet should reference #193

Comment (IanWyszynski) [reply firestore-bigquery-export/CHANGELOG.md]: I agree with @jhuleatt here.

Comment (rachelsaunders) [reply firestore-bigquery-export/CHANGELOG.md]: updated directly in file :-)

**Pull request merged**